source/server/overload_manager_impl.cc

```diff
@@ -31,7 +33,9 @@ class ThresholdTriggerImpl : public OverloadAction::Trigger {
 
 } // namespace
 
-OverloadAction::OverloadAction(const envoy::config::overload::v2alpha::OverloadAction& config) {
+OverloadAction::OverloadAction(const envoy::config::overload::v2alpha::OverloadAction& config,
+                               Stats::Scope& stats_scope)
```

Comment:
What is the planned lifetime of of this overload action? I assume the manager will be global? Will actions be global also? Just trying to understand which scope this is going to use. Will it be the global scope? If so so we need an "overload." prefix? Will it be per-listener? (Probably not).

Reply:
Actions will be global like the manager so I think it makes sense to use global scope for stats. I'll add  an "overload." prefix to the stats names.